src/Relude/Lifted/IORef.hs

```diff
@@ -33,6 +36,12 @@ import qualified Data.IORef as Ref (atomicModifyIORef, atomicModifyIORef', atomi
 
 
 -- | Lifted version of 'Ref.newIORef'.
+{-
+
+TODO: newIORef
+not too sure how to give examples on these
```

Comment:
I think something like this should work:

```haskell
>>> ref <- newIORef False
>>> :t ref
...
```

For `readIORef`:

```haskell
>>> ref <- newiORef 42
>>> readIORef ref
```

And so on for `writeIORef` and others. You can use `readIORef` to display current value inside variable `ref`.